modules/3d/include/opencv2/3d.hpp

```diff
@@ -2588,6 +2588,34 @@ class CV_EXPORTS Octree {
     Ptr<Impl> p;
 };
 
+
+/** @brief Loads a point cloud from a file.
+*
+* The function loadPointCloud loads point cloud from the specified file and returns it.
```

Comment:
The function name is redundant here. The description is always placed with function signature in html and source code.